content/features/search-filters.md

```diff
@@ -0,0 +1,40 @@
++++
+title = "Searching on Crunch (just got better)"

```

Comment:
OK, I removed the "three" from the description (so now "With these new search filters...") just because its kind of 2 filters and one improvement to the already-existing context awareness. Also updated title to "Refine your searches on Crunch"